kythe/go/serving/xrefs/xrefs_test.go

```diff
@@ -1330,11 +1330,18 @@ func TestCrossReferences(t *testing.T) {
 
 	if err := testutil.DeepEqual(&xpb.CrossReferencesReply_Total{
 		Definitions: 1,
-		References:  2,
+		RefEdgeToCount: map[string]int64{
+			"%/kythe/edge/ref": 2,
```

Comment:
Get rid of the leading `%` in the counts

Reply:
done